    public void testDestroy(){
        this.testPool.destroy();
        try{
        this.testPool.aquire();
        fail("pool is already closed");
        }catch (Exception e) {
            // TODO: handle exception
        }
        this.testPool.release(new Object());
    }

